Set the correct install name for libraries that are bundled into the
application. This prevents rev-upgrade from considering the libraries to be
broken.

TODO: URL
--- cmake-proxies/cmake-modules/TenacityFunctions.cmake.orig	2023-11-12 21:06:32.000000000 -0600
+++ cmake-proxies/cmake-modules/TenacityFunctions.cmake	2024-03-08 04:31:58.000000000 -0600
@@ -380,7 +380,7 @@
          PROPERTIES
             PREFIX ""
             FOLDER "libraries" # for IDE organization
-            INSTALL_NAME_DIR ""
+            INSTALL_NAME_DIR "@executable_path/../Frameworks"
             BUILD_WITH_INSTALL_NAME_DIR YES
       )
    endif()
